docs/src/main/asciidoc/security-basic-auth-concept.adoc

```diff
@@ -0,0 +1,36 @@
+[id="security-basic-auth-concept"]
+= Basic Authentication
+ 
+HTTP Basic Authentication uses fields in the HTTP header and is the easiest to set up. Also, it is one of the least resource-demanding techniques that enforce access controls to the Web resources without requiring HTTP cookies, session identifiers, or login pages.
+
+In the context of an HTTP request, Basic authentication is a method for an HTTP user agent, such as a web browser, to provide a user name and password when creating a request. In Basic HTTP Authentication, a request contains a header field in the form of `Authorization: Basic <credentials>`, where credentials are the Base64 encoding of a user ID and password joined by a colon as described in the following example.
+
+====
+.Example
+ 
+If the user name is `Alice` and the password is `secret`, the HTTP authorization header looks authorization as `Authorization: Basic  QWxjZTpzZWNyZXQ=`, where `QWxjZTpzZWNyZXQ=` is a Base64 encoded representation of the `Alice:secret` string.
+====
+ 
+The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
+
+Basic Authentication is a well-specified, simple challenge and response scheme that all web browsers and most web servers understand. However, there are a few limitations associated with Basic Authentication, which include:
+
+Credentials are sent as plain text::
++
+--
+It is required to use HTTPS with Basic Authentication to avoid exposing the credentials. However, if the load balancer terminates HTTPS, the risk of exposing credentials as plain text increases when a request is forwarded to Quarkus over HTTP.
+
+Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only, and the credentials are propagated to the next Quarkus endpoint over HTTP.
+--
+
+Credentials are sent with each request::
++
+--
+In Basic Authentication, a username and password need to be sent with each request, which increases the risk of credentials being exposed.
+--
+
+Application complexity increases::
++
+--
+The Quarkus application needs to ensure itself that usernames, passwords, and roles are managed securely, which increases the application complexity.
```

Comment:
```suggestion
The Quarkus application itself needs to validate that   usernames, passwords, and roles are managed securely.
This can introduce significant complexity to the application. 
Depending on the use case, other authentication mechanisms that delegate user, password, and role management to specialized services may be a better choice.
```

Reply:
Done